packages/desktop-gui/src/specs/specs-list.jsx

```diff
@@ -156,6 +165,22 @@ class SpecsList extends Component {
     return spec.hasChildren ? this._folderContent(spec, nestingLevel) : this._specContent(spec, nestingLevel)
   }
 
+  _toggleFocus = () => {
+    this.setState({
+      isFocused: !this.state.isFocused,
+    })
+  }
+
+  _searchPlaceholderText () {
+    const osKey = window.clientInformation['platform'] === 'MacIntel' ? 'Cmd' : 'Ctrl'
+
+    return `Press ${osKey} + F to make quick search...`
```

Comment:
Didn't read this text before - make it simpler.

```suggestion
    return `Press ${osKey} + F to search...`
```